theme/src/main/java/no/nordicsemi/android/common/theme/view/PagerView.kt

```diff
@@ -64,40 +66,70 @@ fun PagerView(
     viewEntity: PagerViewEntity,
     modifier: Modifier = Modifier,
     itemSpacing: Dp = 0.dp,
+    scrollable: Boolean = true,
+    coroutineScope: CoroutineScope = rememberCoroutineScope(),
+    pagerState: PagerState = rememberPagerState(),
```

Comment:
By passing these as parameters we allow user to use already existent ```coroutine scopes``` / ```pager states```. I currently use this in ```Mesh Sniffer``` to listen to when a page is swiped with a ```Launched Effect``` on the ```pager state```. This allows me to dismiss the current ```snack bar```, if there is one. Another alternative would be to pass the ```snack bar``` state to the pager but I think that's not necessary.